model/utils.py

```diff
@@ -259,23 +254,48 @@ def parse_packets(flp, packet_size_B, direction="data"):
     assert direction in dir_opts, \
         f"\"direction\" must be one of {dir_opts}, but is: {direction}"
 
+    folder = path.dirname(flp)
+    temp_file = path.join(folder, "temp_out")
+
+    client_p = 5555 + unfair_idx
+    server_p = 5201 + unfair_idx
+
+    if direction == "data":
+        filter_s = f"\"tcp.srcport == {client_p} && tcp.dstport == {server_p} && tcp.len == 1448\""
+    else:
+        filter_s = f"\"tcp.srcport == {server_p} && tcp.dstport == {client_p}\""
+
+    os.system(" ".join(["tshark", "-r", flp, filter_s, ">>", temp_file]))
```

Comment:
I think that using `subprocess.check_call()` is preferred:
```python3
subprocess.check_call(["tshark", "-r", flp, filter_s, ">>", temp_file])
```

Reply:
Yea I tried to use that at first, but tshark doesn't seem to like the filter arguments being passed in this way. (It's always returning parsing error or some sort) so I changed it to os.system